source/_integrations/androidtv_remote.markdown

```diff
@@ -20,7 +20,7 @@ ha_platforms:
 ha_integration_type: device
 ---
 
-The Android TV Remote integration allows you to control an Android TV and launching apps. For this to work the Android TV device needs to have [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service) which is pre-installed on most devices.
+The Android TV Remote integration allows you to control an Android TV device by sending [commands](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt) and launching apps. For this to work the Android TV device needs to have [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service) which is pre-installed on most devices.
 
 {% include integrations/config_flow.md %}
 
```

Comment:
(This is for line 29, for some reason I cannot leave a comment there)
Use relative paths to link to cast and universal integrations. That way they work correctly for example on https://rc.home-assistant.io/